docs/TextColumns.md

```diff
@@ -0,0 +1,77 @@
+_New in [:octicons-tag-24: 2.7.7](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
+
+**Notice:** As of fpdf2 release 2.7.7, this is an experimental feature. Both the API and the functionality may change before it is finalized, without prior notice.
+
+
+## Text Columns ##
+
+The `FPDF.text_columns()` method allows to create columnar layouts, with one or several columns. Columns will always be of equal width.
+
+Beyond the parameters common to all text regions, the following are available for text columns:
+
+* l_margin (float, optional) - override the current left page margin.
+* r_margin (float, optional) - override the current right page margin.
+
+Only for `FPDF.text_columns()`:
```

Comment:
I think this line could be removed now that there is only a single method `text_columns()`

Reply:
correct.